distributed/distributed.yaml

```diff
@@ -19,6 +19,7 @@ distributed:
   worker:
     multiprocessing-method: forkserver
     use-file-locking: True
+    max-connections: 10     # maximum simultaneous outgoing connections
```

Comment:
Did you have the opportunity to run some IO intensive benchmark/stress test on a "real" cluster (e.g. on GCP) to measure the impact of that setting on the overall completion time of a data bottlenecked set of tasks?

If you do I would be curious to see the empirical arity of the tree structure of the resulting broadcasting for different values of max-connections.

Reply:
No, I haven't yet done any benchmarking.  I'll play a bit on my local machine.  I may get to trying this out on a larger cluster, but that's uncertain.  Instead, I suspect that we will end up changing the default value over time.